pkg/storage/stores/shipper/shipper_index_client.go

```diff
@@ -90,7 +86,7 @@ func (i *indexClient) init(storageClient client.ObjectClient, limits downloads.L
 	}
 
 	if i.cfg.Mode != indexshipper.ModeReadOnly {
-		uploader, err := i.getUploaderName()
+		uploader, err := i.cfg.GetUniqueUploaderName()
```

Comment:
Q: So this name is applied when only handling `tsdb` files. And boltdb index files are not impacted correct? just double checking.

Reply:
I have just copied the code to build the same uploader name from both `boltdb-shipper` and `tsdb`. There should not be any impact to `boltdb-shipper` file names.